packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/avatar.tests.js

```diff
@@ -124,7 +124,56 @@ QUnit.module('ChatAvatar', moduleConfig, () => {
             assert.strictEqual($img.length, 0);
         });
 
-        QUnit.test('img element should have correct alt attribute', function(assert) {
+        QUnit.test('img should have default alt attribute if alt and name is not defined', function(assert) {
+            this.reinit({
+                name: '',
+                url: 'url',
+            });
+
+            const $img = this.$element.find(`.${AVATAR_IMAGE_CLASS}`);
```

Comment:
Let's extract the image element getting to the module level. This approach help avoid code duplication and reuse utility parts.

Reply:
[2675fac](https://github.com/DevExpress/DevExtreme/pull/28237/commits/2675facd58998531cb47d2ec39ebeffd7ba4a6c9)